### PR TITLE
remove break statement

### PR DESCRIPTION
**Pull request by Kohei-Wada**
deleted remove statement because it works without that.

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -50,13 +50,9 @@ int main(void) {
         types.stage = select_stage();
         switch(types.stage) {
             case STAGE1:
```
Comment (fkawa-play) [inline main.c:53]: Thanks.  This is called `fall through` technique to omit `break`. However, this is difficult to understand that whether `break` is explicitly omitted or not.  So, could you add the following comment above this line?

e.g.

```
// Fall through for selecting stages 1 to 5
```

**Review by fkawa-play — CHANGES_REQUESTED** [review]
Thanks for your fix!

**Review by fkawa-play — APPROVED** [review]
LGTM.  Thanks!!

**Comment by fkawa-play** [comment]
Just aligned the indent

**Comment by Kohei-Wada** [comment]
Thank you!


2021年6月3日(木) 13:07 yushiro_1121 ***@***.***>:

**Pull request merged**